setup.py

```diff
@@ -433,6 +440,9 @@ def main():
         pynepath = "${HOME}/.local/python2.7/site-packages"
     libpath = abspath(joinpath(pynepath, '..', '..', '..'))
     binpath = abspath(joinpath(libpath, '..', 'bin'))
+    copy_ensdf_executables(pynepath)
+    print(pynepath)
+    print(libpath)
```

Comment:
Sorry, those were in to help me debug what was going on with the failing tests.  I'll make sure to remove them as soon as I get the tests passing.